### DISCUSSION TITLE
Search Authors automatically when searching

### DISCUSSION THREAD

**Discussion by kanjieater** [gotson/komga#1519] [Ideas]
### Describe your suggested feature

Noticed AudiobookShelf searches authors at the same time as searching for books. I think it's be useful to be able to search by author without having to use the `author:*` syntax.

### Other details

![image](https://github.com/gotson/komga/assets/32607317/2b181656-127b-4bc2-91e2-b5377383c3a3)

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to the latest version.
- [X] I will fill out all of the requested information in this form.

**Comment by zpaolo11x** [comment]
Then why not make it search in all metadata? Or give the user an option to limit "un-tokened search" to title, title + authors, all metadata except abstract, everything ;)

**Comment by wyldphyre** [comment]
I’ve asked for this before. It would be great for common search scenarios to just work without having to know a certain syntax, and I think Title and Author would be two pretty common things to search for.

**Comment by perelin** [comment]
A very lean thing to do here would be to provide syntax help/tips below/near the search box. Im could see myself getting my hands dirty, would a PR be welcome?

**Reply by gotson** [reply]
The current full text search is most likely going away soon, to be replaced by something else, so don't waste time on this.

**Comment by floatiepen** [comment]
I would love this!  I often include the author in the title just so it comes up in search.  After googling, I found I could search by author:* but that is so tedious.
